myuw/dao/enrollment.py

```diff
@@ -9,6 +9,18 @@
 from myuw.dao.term import get_current_quarter
 
 
+CLASS_CODES = {
+    "FRESHMAN": 1,
+    "SOPHMORE": 2,
+    "JUNIOR": 3,
+    "SENIOR": 4,
+    "GRADUATE": 5,
+}
+
+
+DEFAULT_CLASS_CODE = 6
```

Comment:
Those are just numerical values we can use for sorting.  6 is just bigger than the highest value in the dictionary above.  If we add more defined sort orders, that value would need to change.  I could change it to len(CLASS_CODES.keys()+1, if that would help